model/adjuster/span_tag_to_process.go

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2019 The Jaeger Authors.
+// Copyright (c) 2017 Uber Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package adjuster
+
+import (
+	"github.com/jaegertracing/jaeger/model"
+)
+
+var spanTagsToMove = map[string]struct{}{
+	"otel.library.name": {},
```

Comment:
Resource attributes are the primary targets here

Reply:
I see them in the [spec](https://github.com/open-telemetry/semantic-conventions/blob/main/model/resource/process.yaml), should this be parsed at build time to populate above map or should I add some of the main culprits manually, taken from [here](https://github.com/open-telemetry/opentelemetry-go/blob/main/semconv/v1.21.0/resource.go) for example, since the map itself is a temporary solution to an external problem.

Reply:
Yes, the `process` semconv certainly belongs to Span.Process.

Taking a step back though, I think it's weird that we have to do this translation *after* OTLP data was already translated into Jaeger - I think it's that translation that should be fixed.

Reply:
Will take a look [here](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/translator/jaeger/traces_to_jaegerproto.go).

This PR however is more of dirty/quick fix utility that could be adapted for more general use as suggested [above](https://github.com/jaegertracing/jaeger/pull/4841#issuecomment-1762819773).